### PR TITLE
✨ feat: Mentions Component Token

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
https://github.com/ant-design/ant-design/issues/41884

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Mentions Component Token      |
| 🇨🇳 Chinese |     Mentions Component Token      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at c8c191e</samp>

This pull request adds a new feature to the mentions component that allows users to customize the dropdown menu colors with component tokens. It also updates the style file, the demo pages, and the documentation pages for both English and Chinese languages to reflect this feature.

### <samp>🤖 Generated by Copilot at c8c191e</samp>

* Add the component token demo for the mentions component, which allows customizing the dropdown menu colors using the `ConfigProvider` component ([link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-2bb030ddd3a7c3f0b885545e03969a90d8474243a315d79b5e91ccf0be1c28abR1-R7), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-6247e1bda6388d38c0aa31df57d406970058b590af1ad4f1339f46a74408c462R1-R28), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-96a9c262af881cf703c4bf68a639b92258475cb96538e83a00c45e104c5de608R46), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-9317d359f2ec4a8bc96b985d1d89f2f04251e873847e74be6e1393cc83064a9aR47))
* Update the mentions style file to support the component token properties for the dropdown menu colors, and assign them default values based on the `token.colorBgContainer` value ([link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-9600e9fcd22b061804bbc1129620d20434a9c89a29b42a202c824aadda4c6eceL10-R12), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-9600e9fcd22b061804bbc1129620d20434a9c89a29b42a202c824aadda4c6eceR18-R19), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-9600e9fcd22b061804bbc1129620d20434a9c89a29b42a202c824aadda4c6eceR223-R224))
* Update the migration guide for less variables to include the mentions section, with a table showing the mapping between the less variables and the component tokens for the mentions component ([link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-999566baa64628c49813370e7cb13c3c8f5e0c99ef8fe9cec7c5052985c10d9aL69-R76), [link](https://github.com/ant-design/ant-design/pull/42341/files?diff=unified&w=0#diff-dff3e0a0a5fae848621e55da4e2adaf0ec6c57d5e29d3f601799c8898c280739L69-R76))

### CODE REVIEW

File: components/mentions/style/index.ts [hunk 1]
```diff
@@ -7,14 +7,16 @@ import {
...
 import type { FullToken, GenerateStyle } from '../../theme/internal';
 import { genComponentStyleHook } from '../../theme/internal';
-import { resetComponent, textEllipsis } from '../../style';
 
 export interface ComponentToken {
   zIndexPopup: number;
   dropdownHeight: number;
   controlItemWidth: number;
+  dropdownBg: string;
+  dropdownMenuItemHoverBg: string;
```
Comment (MadCcc) [inline components/mentions/style/index.ts:19]:
```suggestion
  itemHoverBg: string;
```

Comment (MadCcc) [reply components/mentions/style/index.ts:19]: 新增的 token 也要用到 style 里哈

**Comment by MadCcc** [comment]
image
没有变化

**Comment by heiyu4585** [comment]
#42711

**Pull request closed without merging**